### ISSUE TITLE
Radio buttons are freaking out on safari

### ISSUE THREAD

**Issue by mattiasewers** [react-toolbox/react-toolbox#49]
http://gfycat.com/ThoughtfulEachIndianspinyloach

**Comment by javivelasco** [comment]
Apparently there are some rendering problems because this seems to be related with #33

**Comment by TheDeveloperXYZ** [comment]
Seems to be ripple effect glitch.

**Comment by javivelasco** [comment]
Yes but there is no ripple in the time picker and it's  having a glitch that has to be related... Weeeird

**Comment by nathanmarks** [comment]
@javivelasco I cannot reproduce this in the most recent version of OS X and Safari

**Comment by javivelasco** [comment]
Me neither... it's quite weird but I'm almost sure that this kind of rendering error is happening because of `z-index`. #33 was related to it and it looked as a weird rendering error. If somebody gets to reproduce it, please try changing a bit z-indexes to specify the layers order.

**Comment by beata** [comment]
I got the same problem on iOS8.9 Safari.
It happens on every click.

**Comment by javivelasco** [comment]
I'm going to investigate a bit to try to reproduce the issue and fix it. I'm quite sure it's a matter of indexes. Thanks for reporting again!
